pmlc/dialect/op_lib/BUILD

```diff
@@ -27,10 +27,55 @@ PLATFORM_COPTS = select({
     ],
 })
 
+plaidml_cc_library(
+    name = "OpModel",
```

Comment:
Hmm, well so far we've been using the filenaming style of `foo_bar` instead of `FooBar`. I know LLVM/MLIR uses the latter standard. I'm not sure if we should use that style or not, but I'm pretty sure we should be consistent. Maybe we should stick with the old style for now and then convert everything to the new style as one big consolidated commit.